jdbc/src/test/java/software/amazon/timestream/jdbc/TimestreamDatabaseMetaDataTest.java

```diff
@@ -52,6 +52,59 @@ void init() throws SQLException {
     dbMetaData = new TimestreamDatabaseMetaData(mockConnection);
   }
 
+  /**
+   * Checks that an empty result set is returned for getCatalogs
+   */
+  @Test
```

Comment:
Do these newly added test run as part of the GitHub CI Action - https://github.com/awslabs/amazon-timestream-driver-jdbc/actions/runs/3707649792/jobs/6344038925. If these tests were already run, wondering how I can check the test logs  reported from the CI action

Reply:
Download the [test log zip](https://github.com/awslabs/amazon-timestream-driver-jdbc/suites/9895497063/logs?attempt=2) file. Find the file “4_Build with Maven.txt” It has test result of TimestreamDatabaseMetaDataTest.

 

2022-12-20T06:47:52.1273391Z [INFO] Running software.amazon.timestream.jdbc.TimestreamDatabaseMetaDataTest
2022-12-20T06:47:53.0289341Z [INFO] Tests run: 34, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.899 s - in software.amazon.timestream.jdbc.TimestreamDatabaseMetaDataTest